### ISSUE TITLE
Uncaught TypeError: Cannot read property 'disconnect' of undefined

### ISSUE THREAD

**Issue by dmitry-shulaykin** [fand/veda#13]
When I try to toggle it off, I get this error. And after I want to toggle it on, the same error appears again
**Atom**: 1.19.2 x64
**Electron**: 1.6.9
**OS**: Microsoft Windows 10 Enterprise 2016 LTSB
**Thrown From**: [glsl-livecoder](https://github.com/fand/glsl-livecoder) package 0.7.1

### Stack Trace

Uncaught TypeError: Cannot read property 'disconnect' of undefined

```
At C:\Users\dima-pc\.atom\packages\glsl-livecoder\lib\audio-loader.js:62

TypeError: Cannot read property 'disconnect' of undefined
    at AudioLoader.stop (/packages/glsl-livecoder/lib/audio-loader.js:62:18)
    at ThreeShader.stop (/packages/glsl-livecoder/lib/three-shader.js:189:23)
    at GlslLivecoder.stop (/packages/glsl-livecoder/lib/glsl-livecoder.js:104:17)
    at GlslLivecoder.toggle (/packages/glsl-livecoder/lib/glsl-livecoder.js:90:14)
    at HTMLElement.glslLivecoderToggle (/packages/glsl-livecoder/lib/index.js:62:47)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (~/AppData/Local/atom/app-1.19.2/resources/app/src/command-registry.js:265:35)
    at CommandRegistry.handleCommandEvent (~/AppData/Local/atom/app-1.19.2/resources/app/src/command-registry.js:3:65)
    at CommandRegistry.module.exports.CommandRegistry.dispatch (~/AppData/Local/atom/app-1.19.2/resources/app/src/command-registry.js:166:25)
    at AtomEnvironment.module.exports.AtomEnvironment.dispatchContextMenuCommand (~/AppData/Local/atom/app-1.19.2/resources/app/src/atom-environment.js:1344:34)
    at EventEmitter.outerCallback (~/AppData/Local/atom/app-1.19.2/resources/app/src/application-delegate.js:347:31)
    at emitThree (events.js:116:13)
    at EventEmitter.emit (events.js:194:7)
```

### Commands

```
     -4:43.8.0 intentions:highlight (input.hidden-input)
     -4:43.3.0 glsl-livecoder:watch-shader (input.hidden-input)
     -4:41.3.0 intentions:highlight (input.hidden-input)
     -4:40.9.0 glsl-livecoder:watch-shader (input.hidden-input)
     -4:38.7.0 intentions:highlight (input.hidden-input)
     -4:38 glsl-livecoder:watch-shader (input.hidden-input)
     -4:36 intentions:highlight (input.hidden-input)
  2x -4:35.6.0 glsl-livecoder:watch-shader (input.hidden-input)
     -4:20.6.0 intentions:highlight (input.hidden-input)
  4x -4:20.1.0 glsl-livecoder:watch-shader (input.hidden-input)
     -4:13.1.0 intentions:highlight (input.hidden-input)
  2x -4:12.7.0 glsl-livecoder:watch-shader (input.hidden-input)
     -2:49.3.0 glsl-livecoder:toggle (div.line.cursor-line)
     -2:43.7.0 command-palette:toggle (input.hidden-input)
     -2:40.7.0 core:confirm (input.hidden-input)
     -2:40.7.0 glsl-livecoder:toggle (input.hidden-input)
```

### Non-Core Packages

```
atom-beautify 0.30.3 
autocomplete-glsl 0.2.3 
busy-signal 1.4.3 
gcc-make-run 0.2.9 
glsl-livecoder 0.7.1 
intentions 1.1.5 
language-glsl 2.0.1 
linter 2.2.0 
linter-gcc 0.7.1 
linter-glsl 2.1.3 
linter-ui-default 1.6.7 
```

**Comment by fand** [comment]
Thank you for reporting! maybe it's caused by v0.7.1 ...

**Comment by fand** [comment]
I fixed error handling around camera/audio inputs in 50203bc694d7aeb1bf9d0abfc96091a05a5080ff and published v0.7.2.

I couldn't reproduce the bug, so please try v0.7.2 and report again if you still see the errors. 🙏 
@dmitriyqq

**Comment by dmitry-shulaykin** [comment]
I update to 0.7.2 and now toggle works fine
